app/controllers/content_types/school_home_pages_controller.rb

```diff
@@ -0,0 +1,62 @@
+#
+# ContentTypes::SchoolHomePagesController
+#
+# This controller models the content type used by the Law School in Cascade. This content type
+# and its template are in the process of being deprecated but still need to be supported.
+#
+# For more information on the ContentType controller pattern, see the ModularController.
+#
+module ContentTypes
+  class SchoolHomePagesController < ApplicationController
+    layout false
+    before_action :link_static_assets
```

Comment:
This new controller is the important part of this pull request. This method just copies files over from the new `static` directory but gives us a way to effectively track and modify non-modular templates in Cascade Apps.